elf/linker-script.cc

```diff
@@ -348,6 +348,14 @@ void parse_version_script(Context<E> &ctx, std::string path) {
     SyntaxError(ctx, tok[0]) << "trailing garbage token";
 }
 
+template <typename E>
+void parse_dynamic_glob(Context <E> &ctx, std::string_view arg, bool is_cpp) {
+  if (arg == "*")
+    ctx.default_version_from_glob = VER_NDX_GLOBAL;
```

Comment:
I believe linker scripts are parsed after `parse_nonpositional_args`, so isn't it too late to assign a value to this variable?

Reply:
I changed their parsing order in https://github.com/rui314/mold/pull/588/commits/0525da2c0385a5498e8b66c2df121737a417a863.

Reply:
(Or did you actually meant linker script not version script? IIRC linker scripts are not implemented yet?)

Reply:
Linker script is supported, though we support only a limited set of directive such as `INPUT`, `OUTPUT_FORMAT` or `VERSION`. And with `VERSION`, you can embed a version script into a linker script.

Reply:
OK, I see, will try to get this fixed.

Reply:
I've reverted to parsing these as positional args in the latest revision.